src/configuration.ts

```diff
@@ -120,8 +120,83 @@ export function loadConfigurationFromPath(configFilePath: string) {
         let fileData = fs.readFileSync(configFilePath, "utf8");
         // remove BOM if present
         fileData = fileData.replace(/^\uFEFF/, "");
-        return JSON.parse(fileData);
+        let config = JSON.parse(fileData) as IConfigFile;
+
+        let parentConfigNames = config.extends;
+        if (parentConfigNames) {
+            const configFileDir = path.dirname(configFilePath);
+            if (Array.isArray(parentConfigNames)) {
```

Comment:
Yup. Should have done that. Was trying to get the typing work and then forgot about this.